com.unity.multiplayer.mlapi/Runtime/Messaging/RpcQueue/QueueHistoryFrame.cs

```diff
@@ -28,7 +28,7 @@ public enum QueueFrameType
 
 
         public PooledBitReader queueReader;
-
+        private NetworkingManager                        m_NetworkingManager;
```

Comment:
Again, I'd like to see if we can break the need to have PooledBitReaders be a property of a networking manager

Reply:
Is the idea we move them out and use some sort of memory pool or something just say hey give me some bytes or give me these bytes and read/write from them ? 

I agree in having these Pools per Manager make no sense because it will just end up being an allocation that is per manager which could be shared across managers as they end up being short lived objects anyway.

Reply:
Also, @NoelStephensUnity, there was performance sensitivity IIRC in having certain elements (e.g. the network channel string) in the queue frame.  Do you see this as an issue?

If so, to @MFatihMAR's point, we might want to instead store an index to the NM and then look it up in a table

Reply:
Ok, was able to factor this out too